lib/evmone/instructions.hpp

```diff
@@ -741,14 +741,14 @@ Result sstore(StackTop stack, int64_t gas_left, ExecutionState& state) noexcept;
 /// Internal jump implementation for JUMP/JUMPI instructions.
 inline code_iterator jump_impl(ExecutionState& state, const uint256& dst) noexcept
 {
-    const auto& jumpdest_map = state.analysis.baseline->jumpdest_map;
-    if (dst >= jumpdest_map.size() || !jumpdest_map[static_cast<size_t>(dst)])
+    if (dst[3] != 0 || dst[2] != 0 || dst[1] != 0 ||
```

Comment:
Please add a comment on what it does with these !=0s

Reply:
good now?